scripts/generate_rules/generate_package_description.py

```diff
@@ -197,11 +197,10 @@ def generate_short_name(title):
       json.dump(package_description, rule_package_file, indent=2, sort_keys=True)
       print("Rule package file generated at " + str(rule_package_file_path) + ".")
       print("")
-      print("A default query has been generated for each for each rule. Please review each rule in the generated JSON file and:")
+      print("A default query has been generated for each rule. Please review each rule in the generated JSON file and:")
       print(" (1) Add additional queries as required")
       print(" (2) Confirm that the following auto-generated properties are appropriate:")
-      print("    - 'camel_name'.")
```

Comment:
I thought this one could be renamed to `name` and, then, eventually removed it following how it is described in the `help_statement` string around line 25 in this file.